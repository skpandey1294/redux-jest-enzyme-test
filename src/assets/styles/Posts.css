.container1 {
  display: flex;
  flex-wrap: wrap;
  justify-content: center;
}

.card {
  width: 300px;
  height: 300px;
  border-radius: 15px;
  border: 1px #363433 solid;
  margin: 5px;
}
