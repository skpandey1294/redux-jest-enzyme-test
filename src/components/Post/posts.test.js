import React from 'react'
import Posts from './posts'
import { shallow, mount } from 'enzyme'
import configureMockStore from "redux-mock-store";
import { Provider } from 'react-redux'


const mockStore = configureMockStore();
const store = mockStore({});

describe('Testing the Posts Component', () => {

    it('Posts file', () => {

        
        const wrapper = shallow(<Provider store={store}>
            <Posts />
        </Provider>

        )

        expect(wrapper).toMatchSnapshot()

        // console.log(wrapper.dive().find(''))

        // const element = wrapper.find('Posts')

        // console.log(wrapper.debug())
    })
})