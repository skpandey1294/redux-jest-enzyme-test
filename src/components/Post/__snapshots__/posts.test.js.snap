// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the Posts Component Posts file 1`] = `ShallowWrapper {}`;
